<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportSingle</class>
 <widget class="QWidget" name="ExportSingle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Export Location</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Filename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="FileName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Export location</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLineEdit" name="ExportLocation">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BrowseButton">
          <property name="text">
           <string>Browse</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Export Format</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="HTML">
        <property name="text">
         <string>&amp;HTML (preserves formatting)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="PlainText">
        <property name="text">
         <string>&amp;Plain text (discards formatting)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Filename Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="IncludeDate">
        <property name="text">
         <string>Include entry &amp;date in filename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="IncludeJournalName">
        <property name="text">
         <string>Include &amp;journal in filename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="AllowFreeEdits">
        <property name="text">
         <string>Allow custom file&amp;name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
